src/tests/test_openapi.py

```diff
@@ -128,8 +128,8 @@ def test_subpath_view(
                     }
                 ],
                 "related_apis": [
-                    {"type": "WFS", "url": f"{base}/v1/wfs/afvalwegingen/"},
-                    {"type": "MVT", "url": f"{base}/v1/mvt/afvalwegingen/"},
+                    {"type": "WFS", "url": f"{base}/v1/wfs/sub/path/afvalwegingen/"},
```

Comment:
```python
from more_ds.network import URL
BASE = URL(base)
...
{"type": "WFS", "url": BASE / "v1/wfs/sub/path/afvalwegingen/"},
...
```
Not a whole lot of benefit character wise but having one, and only one consistent way of URL construction would be beneficial.